pyverdict/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+import os
+
+setup(name='pyverdict',
+      version='0.0.1',
```

Comment:
Read the version from pom.xml and set it here.

Reply:
It seems that pip doesn't think 0.5.4-SNAPSHOT is good as a version number. 
>UserWarning: The version specified ('0.5.4-SNAPSHOT') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.

Reply:
It won't be a problem since we will release non-snapshot versions only.